SteamList/Controllers/GamesViewController.swift

```diff
@@ -49,8 +68,8 @@ extension GamesViewController: UITableViewDataSource {
     }
     
     func adjustIsFavorite(_ tableView: UITableView,_ indexPath: IndexPath) -> Game {
-        var gameObj = MockData.games[indexPath.row]
         let cell = tableView.cellForRow(at: indexPath)  as! GameTableViewCell
+        var gameObj = games[indexPath.row]
         if cell.cellIcon.imageView?.image == UIImage(systemName: "star") {
```

Comment:
it would be better to write the cell configuration inside of the cell class and access it's method in the viewController

Reply:
done